contrib/automation_tests/test_cases/bottom_up_tab.py

```diff
@@ -45,3 +48,123 @@ def _execute(self):
             raise RuntimeError('First child of the first item ("ioctl") '
                                'of the bottom-up view doesn\'t start with "drm"')
         logging.info('Verified that first child of the first item starts with "drm"')
+
+
+class VerifyBottomUpContentForLoadedCapture(E2ETestCase):
+
+    def _verify_row_count(self, tree_view_table: Table, expected_rows: int):
+        self.expect_eq(tree_view_table.get_row_count(), expected_rows,
+                       "Bottom-up view has {} rows".format(expected_rows))
+
+    def _verify_first_rows(self, tree_view_table: Table, expectations: list[list[str]],
```

Comment:
You can completely ignore this comment, just FYI: Instead of expecting a `list[list[str]]` here, you could relax this to  `Sequence[Sequence[str]]` (with `from typing import Sequence`). This is more generic than `list` and allows for e.g. tuples as well, but still expects the type to be indexable.